### PR TITLE
Port 3193 support update status with git hub actions logs

### PR DESCRIPTION
**Pull request by dvirsegev**

### CODE REVIEW

File: src/__tests__/update-run.test.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import * as core from '@actions/core';
```
Comment (danielsinai) [inline src/__tests__/update-run.test.ts:1]: Let's add another tests that's doing both message + another update, let's say link

File: action.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ inputs:
     description: 'Client secret'
```
Comment (danielsinai) [inline action.yml:14]: Missing extra variables, link, message etc..

File: src/inputs/getInput.ts [hunk 3]
```diff
@@ -24,6 +24,11 @@ const getInput = (): ActionInput => ({
 		required: false,
 	}),
 	runId: core.getInput('runId', { required: false }),
+	message: core.getInput('message', { required: false }),
```
Comment (danielsinai) [inline src/inputs/getInput.ts:27]: I would name it log

**Review by danielsinai — CHANGES_REQUESTED** [review]
🦢 Good job overall, left a few comments
+ Please add it to the README docs

File: src/inputs/getInput.ts (hunk 3 above)
Comment (dvirsegev) [reply src/inputs/getInput.ts:27]: the input or the key we are using?

Comment (danielsinai) [reply src/inputs/getInput.ts:27]: I guess both

File: src/__tests__/update-run.test.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+				runId: 'r_1HDz2pBPYltPcfA1',
+				message: 'Test message',
+			},
+		};
+
+		setInputs(input);
+
+		await main();
+
+		expect(outputMock).toHaveBeenCalledWith('runId', expect.any(String));
```
[the remark this answers is not in the document]
Comment (danielsinai) [reply src/__tests__/update-run.test.ts:36]: Why can't we add one?

**Review by danielsinai — APPROVED** [review]
🦢 
Awesome!!

**Review by danielsinai — APPROVED** [review]
🦢 
Awesome!!

**Pull request merged**